python_modules/libraries/dagster-k8s/tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{39,38,37,36}-{unix,windows}-{default,old_kubernetes}
+envlist = {default,old_kubernetes}
```

Comment:
[1] here we have to list the factors out since `default` here and `papermill2` below are not referenced directly in the tox file